apis/druid/v1alpha1/druid_types.go

```diff
@@ -24,6 +24,16 @@ import (
 // EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
 // NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
 
+// AdditionalContainer defines the additional sidecar container
+type AdditionalContainer struct {
+	// List of configurations to use which are not present or to override default implementation configurations
+	// +optional
+	Configs   map[string]string       `json:"configs,omitempty"`
+	Args      []string                `json:"args,omitempty"`
+	User      int64                   `json:"user,omitempty"`
```

Comment:
let keep it to container security context, let users add the specs.

Reply:
i agree